lib/cdo/safe_browsing.rb

```diff
@@ -36,8 +36,22 @@ def self.determine_safe_to_open(url_to_check)
       response = http.start {http.request(req)}
     end
 
+    threat_type = "Determined safe"
+
     response_value = response.nil? ? nil : JSON.parse(response.body).empty?
 
+    # Determine safe if bad request (response code is 400) or if rate-limited (429)
+    error_response = response.nil? ? false : (response.code == '400' || response.code == '429')
```

Comment:
This variable is a little confusing too, since it's not any error response, just 400 and 429. I think it would be more clear to just check these codes in two elsif cases below.